src/main/webapp/app/controllers/submission/submissionListController.js

```diff
@@ -217,6 +218,15 @@ vireo.controller("SubmissionListController", function (NgTableParams, $controlle
         var packagers = PackagerRepo.getAll();
         var controlledVocabularies = ControlledVocabularyRepo.getAll();
         var submissionTypeList = FieldValueRepo.findValuesByPredicateValue('submission_type');
+        var graduationSemesters = FieldValueRepo.findValuesByPredicateValue('dc.date.issued', function (a, b) {
+            const [monthA, yearA] = a.split(' ');
+            const [monthB, yearB] = b.split(' ');
+
+            const dateA = new Date(`${monthA} 1, ${yearA}`);
+            const dateB = new Date(`${monthB} 1, ${yearB}`);
+
+            return dateB - dateA;
```

Comment:
This makes a lot of assumptions on the date received.
I hope that is not a problem.

The last line:
```
return dateB - dateA;
```

Is likely a problem.
The `new Date()` functions can return `Invalid Date`.
Performing the math on this may result in `NaN`.

I don't know how the filter works when an `NaN` is encountered.

Reply:
It most certainly could be a problem for anything other than the specific date format for graduation semester. I will add some conditions to make it safer and only applicable to the use case.

Reply:
Sort method has been updated to avoid bad input and invalid dates.